src/app/GameController.ts

```diff
@@ -1,65 +1,77 @@
-// import { time } from 'console';
 import { AnimalNames } from '../Enums/AnimalNamesEnum';
-import { Player } from '../Player';
-import { BreedProcessor } from './BreedProcessor';
-import { Bank } from './logic/Bank';
-import { Timer } from './Timer';
+import { GameProcessor } from './logic/GameProcessor';
 import { View } from './View';
+import { Game } from './logic/Game';
+import { defaultGameConfiguration } from './logic/defaultGameConfiguration';
 
 export class GameController {
-  private currentPlayer: Player | undefined;
-  private player: Player | undefined;
-  private timer: Timer;
-  private breedProcessor: BreedProcessor;
+  private game: Game;
+  private gameProcessor: GameProcessor;
   constructor(private view: View) {
-    this.timer = new Timer();
-    this.breedProcessor = new BreedProcessor(new Bank());
+    this.game = new Game(defaultGameConfiguration);
```

Comment:
In the future, the game configuration should be passed in as an argument, that would give the possibilty to modify settings.